lib/components/Upload.js

```diff
@@ -293,6 +343,28 @@ export default class Upload extends Component {
     );
   }
 
+  renderMedtronicSerialNumberInput() {
+    const { upload } = this.props;
+    if (_.get(upload, 'source.driverId', null) !== 'Medtronic') {
+      return null;
+    }
+
+    return (
+      <div>
+        <div className={styles.textInputWrapper}>
+          <p>Enter your 6 digit serial number found on the back of your pump.</p>
+          <input
+            type="number"
```

Comment:
A strange side-effect of using the `number` input type to limit input on this field is that 'e' is actually a valid input due to scientific notation. Not sure if it'll come up in the field, but we might need to do some additional filtering/validation on the handler for the input update.